testsuite/forge-cli/src/main.rs

```diff
@@ -442,6 +437,25 @@ fn single_test_suite(test_name: &str) -> Result<ForgeConfig<'static>> {
                 10000,
                 Some(Duration::from_secs(240)),
             )),
+        "account_creation_state_sync" => config
```

Comment:
@igor-aptos - Do you want to create a test for mix transaction set up as well?

Reply:
Yes,  separate PR.

How do I make this one run in continuous?